pimd/pim_zlookup.c

```diff
@@ -502,9 +502,9 @@ int pim_zlookup_sg_statistics(struct channel_oil *c_oil)
 	if (PIM_DEBUG_ZEBRA) {
 		more.src = *oil_origin(c_oil);
 		more.grp = *oil_mcastgrp(c_oil);
-		zlog_debug(
-			"Sending Request for New Channel Oil Information%pSG VIIF %d(%s)",
-			&more, *oil_parent(c_oil), c_oil->pim->vrf->name);
+		zlog_debug("Sending Request for New Channel Oil Information%pSG VIIF %d(%s:%s)",
+			   &more, *oil_parent(c_oil), ifp->name,
```

Comment:
We have a check below `if (!ifp)` and here we use `ifp->name`. Should we print this after checking?